### ISSUE TITLE
Unicode issue

### ISSUE THREAD

**Issue by morialo3** [savely-krasovsky/waybar-updates#25]
Recently extra Unicode characters get parsed from pacman updates, I mean `\u001b` and `[` followed by numbers and letters
```
{"text":"11","alt":"pending-updates","tooltip":"\u001b[0;1mlibwireplumber \u001b[1;32m0.5.0-2.1\u001b[0m -> \u001b[1;32m0.5.1-1.1\u001b[0m\n\u001b[0;1mqt6-translations \u001b[1;32m6.6.2-1\u001b[0m -> \u001b[1;32m6.6.3-1\u001b[0m\n\u001b[0;1mqt6-webengine \u001b[1;32m6.6.2-1\u001b[0m -> \u001b[1;32m6.6.3-1\u001b[0m\n\u001b[0;1mtelegram-desktop \u001b[1;32m4.15.6-1\u001b[0m -> \u001b[1;32m4.15.6-2\u001b[0m\n\u001b[0;1mwireplumber \u001b[1;32m0.5.0-2.1\u001b[0m -> \u001b[1;32m0.5.1-1.1\u001b[0m
```
Which manifest as the image below.
![240331_22h00m08s_screenshot](https://github.com/L11R/waybar-updates/assets/97380836/3bb42eaa-894c-4cc2-8634-c7edbf748c68)

compared to updates from the AUR, which doesn't have the escape as described above, and looks like this not affected
```
\nprayer-times 0.2.0-1 -> 0.3.0-1\npython-spotdl 4.2.5-1 -> 4.2.5-2\nhyprshade-git 3.2.1.r94.g8110304-1\nlutris-git 0.5.16.r469.g42b74a51c-1\nryujinx-git r3265.c94a73ec60-1","class":"pending-updates"}
```
![240331_22h07m22s_screenshot](https://github.com/L11R/waybar-updates/assets/97380836/a3937574-12c9-4f23-a549-87e1f2b7642a)

This change appeared very recent, I wonder if this issue only on my side or is it general, and needs a solution.
Thanks in advance.

**Comment by morialo3** [comment]
I'll close this since I think this an issue with my notification client not being able to deal with ANSI color outputs.

**Comment by savely-krasovsky** [comment]
I will reopen because we can strip color symbols from messages to be compatible.

**Comment by savely-krasovsky** [comment]
Can someone test this patch?
Replace line 165 with this:
```bash
json $total_updates_count "pending-updates" "$(echo -e "$tooltip" | head -n "$packages_limit" | sed 's/\x1B\[[0-9;]\{1,\}[A-Za-z]//g')" "pending-updates"
````

**Comment by savely-krasovsky** [comment]
I've released `v0.3.1`, please check, nothing should break.
